README.md

```diff
@@ -88,15 +88,15 @@ pip install --extra-index https://pypi.chia.net/simple/ --editable .[dev]
 Then you need to setup the simulator, run the wallet, and create a DID:
 ```
 chia init
-cdv sim create
+chia dev sim create
 export CHIA_ROOT=~/.chia/simulator/main/
 chia start wallet
 chia wallet did create
 ```
 
 Now, you can run the tests:
 ```bash
-pytest tests/test_mint.py
+pytest tests/test_mint.py -n 0
```

Comment:
If we don't work with pytest-xdist then we shouldn't install it.  If we want to make sure it doesn't get used even if something else installs it then we should disable it in the `pytest.ini`.